rafs/src/fs.rs

```diff
@@ -910,7 +910,7 @@ impl FileSystem for Rafs {
         _flags: u32,
     ) -> Result<(Option<Self::Handle>, OpenOptions)> {
         // Cache dir since we are readonly
-        Ok((None, OpenOptions::CACHE_DIR))
+        Ok((None, OpenOptions::CACHE_DIR | OpenOptions::KEEP_CACHE))
```

Comment:
KEEP_CACHE flag does not affect dir?
FYI
```
    /*
     *  Can be filled in by open. It signals the kernel that any
     *  currently cached file data (ie., data that the filesystem
     *  provided the last time the file was open) need not be
     *  invalidated. Has no effect when set in other contexts (in
     *  particular it does nothing when set by opendir()).
     */
    unsigned int keep_cache:1;
```

Reply:
From nydusctl queried fsstats metrics. I never saw opendir ops increased

```
Read Latency:   <1ms    ~20ms   ~50ms   ~100ms  ~500ms  ~1s     ~2s     2s~
Reads Count:    0       0       0       0       0       0       0       0

FOP Counters:
Getattr(1) Readlink(0) Open(0) Release(0) Read(0) Statfs(0) Getxattr(0) Listxattr(0)
Opendir(0) Lookup(13) Readdir(3) Readdirplus(2) Access(0) Forget(0) BatchForget(0)
```

Reply:
> From nydusctl queried fsstats metrics. I never saw opendir ops increased
> 
> ```
> Read Latency:   <1ms    ~20ms   ~50ms   ~100ms  ~500ms  ~1s     ~2s     2s~
> Reads Count:    0       0       0       0       0       0       0       0
> 
> FOP Counters:
> Getattr(1) Readlink(0) Open(0) Release(0) Read(0) Statfs(0) Getxattr(0) Listxattr(0)
> Opendir(0) Lookup(13) Readdir(3) Readdirplus(2) Access(0) Forget(0) BatchForget(0)
> ```

IIUC, only the hybrid mode and passthroughfs-only mode disable no_opendir.